Changelog.md

```diff
@@ -7,7 +7,9 @@ Language Features:
 Compiler Features:
  * Commandline Interface: Return exit code ``2`` on uncaught exceptions.
  * Commandline Interface: Add `--no-cbor-metadata` that skips CBOR metadata from getting appended at the end of the bytecode.
- * EVM: Basic support for the EVM version "Paris".
+ * Support for the EVM version "Paris".
+ * Introduce ``block.prevrandao`` in Solidity and ``prevrandao()`` in inline assembly for EVM versions >= paris.
+ * Deprecate ``block.difficulty`` and disallow ``difficulty()`` in inline assembly for EVM versions >= paris. The change is due to the renaming introduced by [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399).
```

Comment:
Might be good to keep a category like ``EVM`` (even though that's a bit weird), just for keeping these grouped - since the Changelog is sorted alphabetically (we should generally try to keep it sorted alphabetically and will at the latest sort it before the actual release).

Reply:
Added